packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -39,14 +39,13 @@ const defaultMuiThemeOptions: ThemeOptions = {
             transition: '0.2s',
           },
         },
-        text: ({ theme }) => ({
+        text: {
           '&:hover': {
             textDecoration: 'underline',
             textUnderlineOffset: '4px',
-            textDecorationColor: theme.palette.primary.main,
```

Comment:
Removing so that the textDecorationColor will align with the text button color, e.g. when hovering on an "error" text button, the textDecorationColor should also be the "error" color

Reply:
If we need this style, we could use something like `theme.palette[ownerState.color || 'primary']['main']`, but removing as you have done is ideal if it's really unnecessary.

Reply:
Oh great, thanks! I was looking for something like that. I'll add it back for now, in case it is needed.